.travis.yml

```diff
@@ -47,3 +51,6 @@ after_success:
   then
     ./mvnw -s .travis.maven.settings.xml deploy -DskipTests -Dlicense.skip=true;
   fi
+- if [[ "$TRAVIS_TAG" != ""]] && [[ "${TRAVIS_BRANCH}" = "master" ]] && [[ "${TRAVIS_PULL_REQUEST}" = "false" ]]; then
+    ./.travis.push.docker.images.sh
+  fi
```

Comment:
No good. Looking at travis output I see syntax errors in the docker script at this if stmt. Here's the output:

```
$ if [[ "$TRAVIS_TAG" != ""]] && [[ "${TRAVIS_BRANCH}" = "master" ]] && [[ "${TRAVIS_PULL_REQUEST}" = "false" ]]; then ./.travis.push.docker.images.sh fi
/home/travis/.travis/job_stages: eval: line 53: conditional binary operator expected
/home/travis/.travis/job_stages: eval: line 53: syntax error near `"${TRAVIS_BRANCH}"'
/home/travis/.travis/job_stages: eval: line 53: `if [[ "$TRAVIS_TAG" != ""]] && [[ "${TRAVIS_BRANCH}" = "master" ]] && [[ "${TRAVIS_PULL_REQUEST}" = "false" ]]; then ./.travis.push.docker.images.sh fi '
```